source/GovUK.Template.Razor.nuspec.erb

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>GovUK.Template.Razor</id>
+        <version><%= GovukTemplate::VERSION %></version>

```

Comment:
Setting the version from the command line in razor_packager.rb will override whatever is in here (even though they are getting set to the same!)